sklearn/linear_model/_sgd_fast.pyx

```diff
@@ -500,7 +499,7 @@ def _plain_sgd(cnp.ndarray[double, ndim=1, mode='c'] weights,
     cdef double sample_weight
     cdef double class_weight = 1.0
     cdef unsigned int count = 0
-    cdef unsigned int train_count = n_samples - validation_mask.sum()
+    cdef unsigned int train_count = n_samples - np.sum(validation_mask)
```

Comment:
What is the reason for this change? It does seem to raise any warning, isn't it?

Reply:
Oh I see, `sum` does not exist for memoryviews.